build.gradle

```diff
@@ -6,14 +6,14 @@ apply plugin: 'signing'
 group = 'com.feedhenry.gitlabshell'
 archivesBaseName = 'gitlab-shell-client'
 sourceCompatibility = 1.6
-version = '4.0.2'
+version = '4.0.4'
```

Comment:
@camilamacedo86 how come this skipped 4.0.3?

Reply:
@davidffrench has another PR that I made before. :-)